kieker-tools/log-replayer/src/kieker/tools/log/replayer/Settings.java

```diff
@@ -31,16 +32,12 @@
 public class Settings { // NOCS, NOPMD does not need a constructor
 
 	@Parameter(names = { "-i",
-		"--input" }, required = true, description = "Input data directory.", converter = FileConverter.class)
-	private File dataLocation;
-
-	@Parameter(names = { "-p",
-		"--port" }, required = true, description = "Output port.", converter = IntegerConverter.class)
-	private Integer outputPort;
-
-	@Parameter(names = { "-h",
-		"--host" }, required = true, description = "Name or IP address of the host where the data is send to.")
-	private String hostname;
+		"--input" }, required = true, variableArity = true, description = "Input data directory.", converter = FileConverter.class)
+	private List<File> dataLocation;
+	
+	@Parameter(names = { "-kiekerMonitoringProperties",
```

Comment:
The first parameter name must be short, e.g., -k or to be compatible with other tools in the Kieker tools -c
The long parameter should be --kieker-monitoring-properties or --configuration or --kieker-configuration or --output-configuration

Reply:
Thanks for the hint, I've changed it to `-c` or `--configuration`, to be compatible with the other Kieker tools.